deployment/ansible/roles/raster-foundry.jenkins/templates/jenkins.conf.j2

```diff
@@ -0,0 +1,18 @@
+server {
+    listen 80;
+    server_name {{ jenkins_server_name }};
+
+    location / {
+
+      proxy_set_header        Host $host:$server_port;
+      proxy_set_header        X-Real-IP $remote_addr;
+      proxy_set_header        X-Forwarded-For $proxy_add_x_forwarded_for;
+      proxy_set_header        X-Forwarded-Proto $scheme;
+
+      # Fix the "It appears that your reverse proxy set up is broken" error.
+      proxy_pass          http://127.0.0.1:{{ jenkins_http_port }};
+      proxy_read_timeout  90;
+
+      proxy_redirect      http://127.0.0.1:{{ jenkins_http_port }} {{ jenkins_server_name }};
```

Comment:
I think that this was added to the Nginx example to help redirect things to HTTPS. In our case, I think we'd want this to be set to `off`.